motionplan/tpSpaceRRT.go

```diff
@@ -286,6 +288,14 @@ func (mp *tpSpaceRRTMotionPlanner) rrtBackgroundRunner(
 		}
 	}
 
+	// check if we start at the goal
```

Comment:
yes, here we make sure that if the goal is the starting position we return early.

Reply:
This isn't really a bug, just someone misusing the package which I don't think we should really be catering to. Would be curious for input from @nicksanford and @biotinker

Reply:
I think that something should ensure that a user telling a robot to go to where it already is (within the plan deviation mm) results in the robot not moving. 

I don't know whether that is a motion service concern or a motion plan package concern but I do think that check should exist somewhere. 

It does seem like it is unrelated to implementing MoveOnMapNew though. @nfranczak could we add this in a later PR given there is not team consensus on how to do this?

Reply:
This seems like a good idea to me

Reply:
> could we add this in a later PR given there is not team consensus on how to do this?
that sounds good to me.
I will create a ticket so that this is tracked.

Reply:
https://viam.atlassian.net/browse/RSDK-6272